src/sentry/static/sentry/app/views/organizationIntegrations/constants.tsx

```diff
@@ -7,3 +7,46 @@ export const colors = {
   [NOT_INSTALLED]: 'gray2',
   [PENDING]: 'yellowOrange',
 };
+
+/**
+ * Integrations in the integration directory should be sorted by their popularity (weight).
+ * These weights should be hardcoded in the application itself.
+ * We can store this in a map where the key is the integration slug and the value is an integer and represents the weight.
+ * The weights should reflect the relative popularity of each integration.
+ */
+
+export const popularityWeights = {
+  // First-party-integrations
+  slack: 50,
+  github: 20,
+  jira: 10,
+  bitbucket: 10,
+  gitlab: 10,
+  pagerduty: 10,
+
+  // Plugins
+  webhooks: 20,
+  trello: 8,
+  heroku: 8,
+  pivotal: 8,
+  twilio: 8,
+  pushover: 8,
+  redmine: 8,
```

Comment:
@NisanthanNanthakumar I really am questioning some of the weights of plugins. Redmine is a marginal plugin we thought about deprecating. How is it higher than Asana? Pushover, Pivotal, Phabricator, and Teamwork all seem too high up. Wondering how @jonesphillip got these numbers.